src/Status/Status.tsx

```
@@ -0,0 +1,175 @@
+import { useEffect, useState } from '@wordpress/element';
+import Menu from './Menu';
+import Tab from './Tab';
+import SettingsParser from './SettingsParser';
+import apiFetch from '@wordpress/api-fetch';
+
+export type StatusElement = {
+    headers?: Array< string >;
+    payload?: Object;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)